backend/src/main/java/com/tourneynizer/tourneynizer/dao/TeamRequestDao.java

```diff
@@ -100,12 +105,12 @@ public TeamRequest requestUser(User requested, Team team, User requester) {
     }
 
     public List<TeamRequest> getRequestsForUser(User user) {
-        String sql = "SELECT * FROM teamRequest WHERE user_id=? AND requester_id<>user_id;";
+        String sql = "SELECT * FROM teamRequest INNER JOIN teams on (teamRequest.team_id = teams.id) WHERE teamRequest.user_id=? AND teamRequest.user_id<>teams.creator_id;";
```

Comment:
Can you also inner join with the user table to return a user object as well

Reply:
I'll leave that up to Porter in a future PR because that is going to break front ends.